packages/clerk-js/src/ui/components/SignUp/SignUpContinue.tsx

```diff
@@ -77,8 +71,25 @@ function _SignUpContinue() {
       placeholder: localizationKeys('formFieldInputPlaceholder__password'),
       validatePassword: true,
     }),
+    __experimental_legalAccepted: useFormControl('__experimental_legalAccepted', '', {
+      type: 'checkbox',
+      label: 'I agree to the Terms of Service and Privacy Policy',
```

Comment:
We should replace with a `localizationKeys` here? 🙃

Reply:
Good catch Niko!

Reply:
Oh to be fair this can removed completely as we have custom component so this is not rendered at all